Gruntfile.js

```diff
@@ -106,6 +106,7 @@ module.exports = function (grunt) {
         options: {
           title: 'Angular-Patternfly Documentation',
           dest: 'dist/docs',
+          imageLink: 'https://rawgit.com/patternfly/patternfly/master/dist/img/logo-alt.svg',
```

Comment:
> Hmm...I don't see this logo anywhere when I do 'grunt ngdocs:view'

Top left corner.



> Anyone care that I am linking out to this or do you want me to add it to the project?

I think we should probably add it unless we are fairly confident that it's not going to be renamed, etc.